pkgs/by-name/py/pysolfc/package.nix

```diff
@@ -0,0 +1,108 @@
+{
+  lib,
+  callPackage,
+  desktop-file-utils,
+  fetchzip,
+  freecell-solver,
+  python3Packages,
+  testers,
+  # Configurable options
+  withFreecellSolver ? true,
+  withPillow ? true,
+  withPygame ? true,
+}:
+
+let
+  _sources = callPackage ./sources.nix { };
```

Comment:
Why not inline this? The sources are not generated and the file also contains extra things like pname

Reply:
> The sources are not generated 

Yet.